src/index.js

```diff
@@ -1,6 +1,6 @@
 import Button from "./Button"
 import AlertBox from "./AlertBox"
 import Card from "./Card"
-import Input from "./Input"
+import SmallNav from "./SmallNav"
```

Comment:
```suggestion
import Input from "./Input"
import SmallNav from "./SmallNav"
```
Input jangan dihapus, itu kodingan ku